dicomanonymizer/simpledicomanonymizer.py

```diff
@@ -34,17 +33,28 @@ def apply_regexp(dataset, tag):
 
 # Default anonymization functions
 
+def get_new_UID(old_uid: str) -> str:
+    """
+    Lookup new UID in cached dictionary or create new one if none found
+    """
+    from pydicom.uid import generate_uid
+    if old_uid not in dictionary:
+        dictionary[old_uid] = generate_uid(None)
```

Comment:
I had 3 options:
1. Don't change the prefix/root UID: This seemed hard, since I could not figure out how to separate prefix from suffix.
2. Use pydicom's default prefix, i.e., use generate_uid(): Valid option
3. Use 2.25 as prefix, i.e., use generate_uid(None): Valid option

I chose option 3 to get a shorter UID. The suffix is generated using uuid.uuid4(), which is 32 Hex=37 digits long. That level of 'uniqueness' seems to be generally acceptable [https://en.wikipedia.org/wiki/Universally_unique_identifier#Version_4_(random)]

What do you recommend?